### PR TITLE
Feature for changes

### PR DESCRIPTION
**Pull request by bernabasy**
in this pull request, I did the following
-  added the index.html file
- added style.css
- added test.MD

### CODE REVIEW

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @bernabasy 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses:

no-we-were-so-close-pauline-turner-brooks


## Highlights
:heavy_check_mark: All linters are passing
:heavy_check_mark: Good PR summary

## Required Changes ♻️

- [ ] You need to have a valid and descriptive readme file. Kindly consider editing the readme template according to your credentials and the project you are working on. You need to edit the `description`, `how to set up the project`, `author`, `issues`, `license` etc. Don't just copy the microverse readme template. It is there to provide guidance. :pray: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,56 +1,23 @@
...
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- html
+- css
 
-## Live Demo (if available)
+-## Getting Started
```
Comment (topeogunleye) [inline README.md]: - [ ]  As suggested by the previous reviewer, please kindly add steps for how to set up the project:

## EXAMPLE

```
## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Prerequisites
Basic knowledge of HTML, CSS.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/bernabasy/hello-microvers-frist-project/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/bernabasy/hello-microvers-frist-project.git
```
```
- Once the files are on your machine, open the _Portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/)

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).
```

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @bernabasy, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
-Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @bernabasy, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by topeogunleye** [comment]
> ### Hi @bernabasy,
> # Status: Approved ✔️
> Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it 💪 Congratulations! 🎉 congratulations [ congratulations ](https://user-images.githubusercontent.com/52934128/163691476-087b4b9c-825b-40a3-85a1-d1bde44ffa53.gif) [ ](https://user-images.githubusercontent.com/52934128/163691476-087b4b9c-825b-40a3-85a1-d1bde44ffa53.gif)
> 
> > 
> 
> ## To highlight 🍾
> * All linters are passing ✔️
> * Followed Github flow ✔️
> * Professional README file. ✔️
> * PR has a good description ✔️
> 
> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag @topeogunleye in your question so I can receive the notification.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**